### ISSUE TITLE
Roaming config?

### ISSUE THREAD

**Issue by MarcDBehr** [drewmccal/anytone-878-northeast-codeplug#1]
Hi Drew,

Appreciate that you posted this info. Do you by chance have any roaming config info? 
A just got the 878 and if I find any missing data, I will submit a pull request.

Marc

**Comment by drewmccal** [comment]
Hi Marc,

Yeah, I haven't experimented with the roaming features yet, but please feel free to help me adapt this code plug.

Drew

**Comment by k2bkr** [comment]
Hi Drew, 
New to this and have the 878 to program. Wondering if you could add to the code plug Long Island NY DMR repeaters. If your cruising I95 on your north south drives, you might hit Long Island. I would like to download and install. 73 de K2BKR

**Comment by drewmccal** [comment]
@k2bkr Yeah, I could do that if you'd like.  Do you have links to some of the associations/repeater groups on LI you like?

**Comment by k2bkr** [comment]
Drew

 

Drew

Thank You and I finally got it going. Great little radio. Now I’m building up 395 >90>2 into Boston. I come up from New London Ferry and travel to Boston. Next year heading into Maine, so Ill be adding to it again.

 

73’s

K2bkr

**Comment by drewmccal** [comment]
@k2bkr Could you push your new codeplug to a different branch or open a pull request?  I'd love to make it available to others.  It would also help me expand the existing codeplug.  Yes, the Anytone is a great radio!

**Comment by k2bkr** [comment]
Drew

 

I will try to do that and send you a link.

 

73’s 

John
